benchmarks/src/main/kotlin/com/google/samples/apps/nowinandroid/foryou/ForYouActions.kt

```diff
@@ -41,6 +43,11 @@ fun MacrobenchmarkScope.forYouWaitForContent() {
  */
 fun MacrobenchmarkScope.forYouSelectTopics(recheckTopicsIfChecked: Boolean = false) {
     val topics = device.findObject(By.res("forYou:topicSelection"))
+    val withChildren = topics.childCount != 0
+    if (!withChildren) {
```

Comment:
```suggestion
    val noChildren = topics.childCount == 0
    if (noChildren) {
```
Maybe use `noChildren` is more readable, or noTopics.